testing/features/support/drivers/browserstack.rb

```diff
@@ -36,7 +36,7 @@ def valid_browserstack_drivers
 
     def browser_type_class
       if specialized_driver?
-        Module.const_get("Drivers::Browserstack::#{browser.to_s.camelize}")
+        Module.const_get("Drivers::Browserstack::#{browser.to_s.pascalize}")
```

Comment:
Just FYI really - you've already got activesupport in your Gemfile so this could be:

```ruby
require 'active_support/core_ext/string'

# ...

Module.const_get("Drivers::Browserstack::#{browser.to_s.camelize(:upper)}")
```

Reply:
Yeh this was causing a few issues. Happy to look into optimising this later. The fork of protobuf-cucumber might also be losing AS soon as well.

The issue here is the AS (or protobuf-cucumber -> AS version), of this method conflicted with this one, and caused the entire show to fail.

Reply:
Makes sense that it conflicted, but of course we can just use the AS method instead of rolling our own.